modules/reindex/src/test/resources/rest-api-spec/test/delete_by_query/20_validation.yaml

```diff
@@ -6,6 +6,19 @@
         index: _all
 
 ---
+"no query fails":
+
+  - skip:
+      version: "6.0.0 - "
```

Comment:
I do not understand the skip, shouldn't it be " - 5.99.99" (or something like that)?